meta-mender-tegra/meta-mender-tegra-jetpack5/templates/bblayers.conf.sample

```diff
@@ -0,0 +1,16 @@
+# POKY_BBLAYERS_CONF_VERSION is increased each time build/conf/bblayers.conf
```

Comment:
@TheYoctoJester are the templates here still used with the kas setup?  If so don't we need to change the layer reference for meta-mender-tegra?